Update instructions (#7)
Agoric/zoe-sketchbook: README.md

An easy way to get started building JavaScript smart contracts on Zoe.
Draft and lightly test your smart contact code here.

## Getting started

> The following instructions are written for MacOS. If you are
> using a different operating system or having any difficulties, please

Reviewer: ```suggestion
The following instructions are written for MacOS. If they don't work for you, or you can contribute instructions for a different platform, please
```
